public/static/docs/tutorials/versioning.md

```diff
@@ -245,7 +245,7 @@ That's it! We have a second model and dataset saved and pointers to them
 committed with Git. Let's now look at how DVC can help us go back to the
 previous version if we need to.
 
-## Switching between versions
+## Switching between model versions
```

Comment:
same here as previous, not exactly correct

Reply:
True. I'm not sure why I only added "models" here... Probably to distinguish from source code (switched with `git checkout`)... ⏳

Reply:
Addressed per https://github.com/iterative/dvc.org/pull/933#pullrequestreview-351862439 above.